src/components/RoomNameInput.js

```diff
@@ -114,7 +111,6 @@ RoomNameInput.defaultProps = defaultProps;
 
 export default compose(
     withLocalize,
-    withFullPolicy,
     withOnyx({
         reports: {
```

Comment:
Might be out of scope of this PR but I'm curious, are these Onyx keys needed in this components?

Reply:
Doesn't look like it. I'll remove those too since its a quick improvement